azure/deploymentframeworks/arm/output.ftl

```diff
@@ -37,19 +37,73 @@
     }]
 [/#function]
 
-[#macro armOutput name type value condition=""]
-    [@mergeWithJsonOutput
-        name="outputs"
-        content=
-            {
-                name : {
-                    "type" : type,
-                    "value" : value
-                } +
-                attributeIfContent("condition", condition)
-            }
-    /]
-[/#macro]
+[#function getArmOutput name type value condition=""]  
+    [#return {
+        name : {
+            "type" : type,
+            "value" : value
+        } +
+        attributeIfContent("condition", condition)
+    }]
+[/#function]
+
+[#function constructArmOutputsFromMappings id name scope mappings=[]]
```

Comment:
This function could do with a bit more commenting to explain the key differences between the output types

Reply:
Yep, have commented on this in another PR - #202